.github/workflows/osx.yml

```diff
@@ -90,6 +90,7 @@ jobs:
           -DCMAKE_CXX_FLAGS="-Wall -Wshadow -pedantic -Werror -Wsign-compare -Wtype-limits -Wignored-qualifiers -Wempty-body -Wuninitialized" \
           -DCMAKE_INSTALL_PREFIX=$PWD/../install \
           -DKokkosKernels_ENABLE_TESTS=ON \
+          -DKokkosKernels_ENABLE_BENCHMARKS=ON \
```

Comment:
we need to double check if kokkos-kernels team wants to actually run this in the CI, but I think the answer is no if I remember correctly from what @lucbv said

Reply:
Indeed we do not want to run them as performance benchmarks are typically taking a lot more time to run and will be marked as failed due to time outs.

Reply:
:+1: d953f8f75e029f5929d4c02c3c3c8319a413cea3